combat-trainer.lic

```diff
@@ -1102,7 +1102,19 @@ class ManipulateProcess
 
   def manipulate
     bput('manip stop all', 'You relax your will', 'But you aren')
-    bput("manipulate friendship #{@filtered_npcs.first}",  'You attempt to empathically manipulate', 'You strain')
+    @manipulate_count = 0
```

Comment:
Should this be a local variable (i.e. drop the `@`)? I don't see it used outside of this method...